chef/cookbooks/glance/recipes/ceph.rb

```diff
@@ -24,35 +27,38 @@
 else
   # If external Ceph cluster will be used,
   # we need install ceph client packages
-  if node[:platform] == "suse" && File.exists?("/etc/ceph/ceph.client.admin.keyring")
+  return unless File.exists?(ceph_conf) && File.exists?(admin_keyring)
```

Comment:
Hm, we also still want support the case where the admin manually creates the ceph users and pools, right? Not everybody will be able to put their admin keyring on the glance node.

So shouldn't we split this somehow? Install the ceph packages when ```File.exists?(ceph_conf)``` and after that exit the recipe if ```admin_keyring``` is not present?